src/app/containers/MediaPlayer/index.jsx

```diff
@@ -1,55 +1,49 @@
 import React from 'react';
-import Helmet from 'react-helmet';
 import pathOr from 'ramda/src/pathOr';
-import Caption from '../Caption';
-import videoMetadata from './metadata';
-import {
-  NestedGridParentLarge,
-  NestedGridParentSmall,
-  NestedGridItemChildSmall,
-  NestedGridItemChildLarge,
-  GridItemConstrainedLargeNoMargin,
-  GridItemConstrainedSmall,
-} from '../../lib/styledGrid';
-
+import styled from 'styled-components';
+import Amp from './Amp';
+import Canonical from './Canonical';
+import Metadata from './Metadata';
+import embedUrl from './helpers/embedUrl';
+import filterForBlockType from '../../lib/utilities/blockHandlers';
+import useToggle from '../Toggle/useToggle';
+import { RequestContext } from '../../contexts/RequestContext';
+import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 import {
   mediaPlayerPropTypes,
   emptyBlockArrayDefaultProps,
 } from '../../models/propTypes';
-import filterForBlockType from '../../lib/utilities/blockHandlers';
-import { RequestContext } from '../../contexts/RequestContext';
-import useToggle from '../Toggle/useToggle';
 
-const selectWrappers = orientation => {
-  const wrapperSpan = {
-    default: '6',
-    group5: '12',
-  };
-  let ParentWrapper = NestedGridParentLarge;
-  let ChildWrapper = NestedGridItemChildLarge;
-  let Container = GridItemConstrainedLargeNoMargin;
-  let portrait = false;
-
-  if (orientation === 'Portrait') {
-    ParentWrapper = NestedGridParentSmall;
-    ChildWrapper = NestedGridItemChildSmall;
-    Container = GridItemConstrainedSmall;
-    wrapperSpan.default = '4';
-    portrait = true;
-  }
-
-  return { ParentWrapper, ChildWrapper, Container, wrapperSpan, portrait };
+const StyledContainer = styled.div`
+  padding-top: ${({ orientation }) =>
+    orientation === 'Portrait' ? '177.78%' : '56.25%'};
+  position: relative;
+  overflow: hidden;
+`;
+
+const placeholderImage = src => {
+  const parts = src.split('/');
+  const [domain, media, imgService, width, ...extraParts] = parts;
+  const definedWidth = width.replace('$width', '512');
```

Comment:
srcset?

Reply:
Yeah this will be added at a later date 🙂